lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -46,21 +47,19 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      */
     constructor(configuration: Configuration) {
         super(configuration);
-        this.initializeBrokering();
     }
 
     /**
      * 
      */
-    private initializeBrokering(): void {
-        if (this.config.system.brokerOptions.actAsBroker) {
+    async initializeBrokering(): Promise<void> {     
```

Comment:
We should add this to `IPublicClientApplication` as well.

Reply:
Done